### ISSUE TITLE
Why is psycopg2 needed?

### ISSUE THREAD

**Issue by heckad** [devind-team/graphene-django-filter#79]
Dependencies includes `psycopg2-binary = "^2.9.3"`.  Why is it needed?

**Comment by thomas-arun** [comment]
Yes, seems like `psycopg2` is not a direct requirement for this package. 
I will test it and update this ticket. Thank you

**Comment by heckad** [comment]
Also it would be grate to relax the `django-filter` version.

**Comment by heckad** [comment]
@thomas-arun, any news?
